scrapers/add_db_entry.py

```diff
@@ -50,6 +51,24 @@
         else:
             data['deceased'] = int(data['deceased'])
 
+        # Parse optional data.
+        rest = match.group(5)
+        extras_match = re.search('# Extras: ([^#]+)', rest)
+        if extras_match.group(1):
```

Comment:
This should be `if extras_match:`, because if there was not match, you'll get `Error: 'NoneType' object has no attribute 'group'`

Reply:
Good catch!